packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -82,6 +92,36 @@ type ScrollIntoViewHandle = {
   scrollIntoView: () => void;
 };
 
+const NavItemContentClickContainer = styled("div", {
+  shouldForwardProp: (prop) =>
+    prop !== "odysseyDesignTokens" &&
+    prop !== "isCompact" &&
+    prop !== "isDisabled",
+})<{
+  odysseyDesignTokens: DesignTokens;
+  isCompact?: boolean;
+  isSelected?: boolean;
+  disabled?: boolean;
+  isDisabled?: boolean;
+}>(({ odysseyDesignTokens, isCompact, isDisabled }) => ({
+  display: "flex",
+  alignItems: "center",
+  width: "100%",
+  minHeight: isCompact ? "37px" : "48px",
```

Comment:
Definitely put these somewhere else because they're used in multiple components.

Reply:
using the tokens instead